tools/changelogger-release.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+. "$BASE/tools/includes/check-osx-bash-version.sh"
+. "$BASE/tools/includes/chalk-lite.sh"
+
+# Print help and exit.
+function usage {
+	cat <<-EOH
+		usage: $0 [-v] [-b] <slug>
+
+		Prepare a release of the specified project and everything it depends on.
+		 - Run \`changelogger write\`
+		 - Run \`tools/project-version.sh\`
+
+		Pass \`-b\` to release pass \`--prerelease=beta\`.
+	EOH
+	exit 1
+}
+
+if [[ $# -eq 0 ]]; then
+	usage
+fi
+
+# Sets options.
+VERBOSE=
+BETA=false
+while getopts ":vbh" opt; do
+	case ${opt} in
+		v)
+			if [[ -n "$VERBOSE" ]]; then
+				VERBOSE="${VERBOSE}v"
+			else
+				VERBOSE="-v"
+			fi
+			;;
+		b)
+			BETA=true
+			;;
+		h)
+			usage
+			;;
+		:)
+			die "Argument -$OPTARG requires a value."
+			;;
+		?)
+			error "Invalid argument: -$OPTARG"
+			echo ""
+			usage
+			;;
+	esac
+done
+shift "$(($OPTIND -1))"
+
+if [[ -z "$VERBOSE" ]]; then
+	function debug {
+		:
+	}
+fi
+
+# Determine the project
+[[ -z "$1" ]] && die "A project slug must be specified."
+SLUG="${1#projects/}" # DWIM
+SLUG="${SLUG%/}" # Sanitize
+if [[ ! -e "$BASE/projects/$SLUG/composer.json" ]]; then
+	die "Project $SLUG does not exist."
+fi
+
+cd "$BASE"
+yarn jetpack install --all
```

Comment:
Should we consider the case where `yarn jetpack` would not be available, or should we let it fail and let the release manage figure things out?

Reply:
`yarn jetpack` should work as long as you ran `yarn install` at some point. I suppose we could add that in here if that's a concern.

Reply:
True. I guess it shouldn't be an issue for the folks running the scripts, since they'll most likely already be familiar with `yarn jetpack`. 

Marking this as resolved.